File: rh-front/src/app/models/absenceRequest.ts
export class AbsenceRequest {
    id : number;
    dateAbs : Date;
    natureAbsence : string;
    justificatif : boolean;
    duree : string;
    EmployeId : number;

    constructor(){
        this.id = 0;
        this.dateAbs = new Date();
        this.natureAbsence = '';
        this.justificatif = false;
        this.duree = '';
        this.EmployeId = 0;
    }
}

File: absence-service/src/main/java/com/example/absenceservice/model/AbsenceRequest.java
package com.example.absenceservice.model;

import com.example.absenceservice.entities.NatureAbsence;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.Date;

@Data @AllArgsConstructor @NoArgsConstructor
public class AbsenceRequest {
    private Long id;
    private Date dateAbs;
    private NatureAbsence natureAbsence;
    private boolean justificatif;
    private String duree;
    private Long EmployeId;
}
